### PR TITLE
feat(meter): added min max value on meter chart

### PR DESCRIPTION
**Pull request by maulik-thaker1**
- Added min max value on meter gauge
- Max value will be the data total value
- Added x-label below needle

### CODE REVIEW

File: src/common/config/chartTypes/meter.js [hunk 1]
```diff
@@ -12,7 +12,6 @@ export const options = () => {
     radius: '80%',
     circumference: 180,
     rotation: 270,
-    aspectRatio: 1.7,
```
Comment (brian-patrick-3) [inline src/common/config/chartTypes/meter.js:15]: It looks weird with the default value of 1, too much white space. Maybe just reduce it slightly, ex ~1.5?

Comment (maulik-thaker1) [reply src/common/config/chartTypes/meter.js:15]: I think 1.4 is good

Comment (maulik-thaker1) [reply src/common/config/chartTypes/meter.js:15]: Done

**Review by brian-patrick-3 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -81,6 +81,7 @@
     "chromatic": "^11.2.0",
     "clean-webpack-plugin": "^4.0.0",
     "css-loader": "^6.7.3",
+    "esbuild": "^0.20.2",
```
[the remark this answers is not in the document]
Comment (maulik-thaker1) [reply package.json:84]: this is suggested by checks (header rules).  Its fail to deploy PR needs **esbuild** package according to our machines. As far as I know netlify requires @esbuild/linux-64 to deploy.

[the remark this answers is not in the document]
Comment (brian-patrick-3) [reply package.json:84]: Ah that's probably just an issue on Netlify's end. Just need to re-run the deploy if it errors out.

[the remark this answers is not in the document]
Comment (maulik-thaker1) [reply package.json:84]: Not helped I re-run several times but not works.

**Pull request merged**